include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,553 @@
+    bool accelerationStructure = false;
+    bool accelerationStructureCaptureReplay = false;
+    bool accelerationStructureIndirectBuild = false;
+    bool accelerationStructureHostCommands = false;
+    bool descriptorBindingAccelerationStructureUpdateAfterBind = false;
+            
+    /* BufferDeviceAddressFeaturesKHR *//* VK_KHR_buffer_device_address *//* MOVED TO Vulkan 1.2 Core */
+
+    /* DynamicRenderingFeaturesKHR *//* VK_KHR_dynamic_rendering *//* MOVED TO Vulkan 1.3 Core */
+
+    /* [!!NV Version below, struct doesn't exist in vk headers] VK_KHR_fragment_shader_barycentric */
+
+    /* FragmentShadingRateFeaturesKHR *//* VK_KHR_fragment_shading_rate */
+    //VkBool32           pipelineFragmentShadingRate;
+    //VkBool32           primitiveFragmentShadingRate;
+    //VkBool32           attachmentFragmentShadingRate;
+    
+    /* GlobalPriorityQueryFeaturesKHR *//* VK_KHR_global_priority */
+    //VkBool32           globalPriorityQuery;
+
+    /* ImagelessFramebufferFeaturesKHR *//* VK_KHR_imageless_framebuffer *//* MOVED TO Vulkan 1.2 Core */
+    /* Maintenance4FeaturesKHR *//* VK_KHR_maintenance4 *//* MOVED TO Vulkan 1.3 Core */
+    /* MultiviewFeaturesKHR *//* VK_KHR_multiview *//* MOVED TO Vulkan 1.1 Core */
+
+    /* PerformanceQueryFeaturesKHR *//* VK_KHR_performance_query */
+    //VkBool32           performanceCounterQueryPools;
+    //VkBool32           performanceCounterMultipleQueryPools;
+
+    /* PipelineExecutablePropertiesFeaturesKHR *//* VK_KHR_pipeline_executable_properties */
+    //VkBool32           pipelineExecutableInfo;
+    
+    /* VK_KHR_portability_subset - PROVISINAL/NOTAVAILABLEANYMORE */
+
+    /* PresentIdFeaturesKHR *//* VK_KHR_present_id */
+    //VkBool32           presentId;
+
+    /* PresentWaitFeaturesKHR *//* VK_KHR_present_wait */
+    //VkBool32           presentWait;
+    
+    /* RayQueryFeaturesKHR *//* VK_KHR_ray_query */
+    bool rayQuery = false;
+
+    /* VK_KHR_ray_tracing !! Replaced/Removed */
+    /* VK_KHR_ray_tracing_maintenance1 *//* added in vk 1.3.213, the SDK isn't released yet at this moment :D */
+
+    /* RayTracingPipelineFeaturesKHR *//* VK_KHR_ray_tracing_pipeline */
+    bool rayTracingPipeline = false;
+    bool rayTracingPipelineShaderGroupHandleCaptureReplay = false;
+    bool rayTracingPipelineShaderGroupHandleCaptureReplayMixed = false;
+    bool rayTracingPipelineTraceRaysIndirect = false;
+    bool rayTraversalPrimitiveCulling = false;
+
+    /* SamplerYcbcrConversionFeaturesKHR *//* VK_KHR_sampler_ycbcr_conversion *//* MOVED TO Vulkan 1.1 Core */
+    /* SeparateDepthStencilLayoutsFeaturesKHR *//* VK_KHR_separate_depth_stencil_layouts *//* MOVED TO Vulkan 1.2 Core */
+    /* ShaderAtomicInt64FeaturesKHR *//* VK_KHR_shader_atomic_int64 *//* MOVED TO Vulkan 1.2 Core */
+
+    /* ShaderClockFeaturesKHR *//* VK_KHR_shader_clock */
+    //VkBool32           shaderSubgroupClock;
+    //VkBool32           shaderDeviceClock;
```

Comment:
```cpp
shaderSubgroupClock = GL_ARB_shader_clock;
shaderDeviceClock = GL_EXT_shader_realtime_clock;
```

Reply:
always enable the base extension, report as limit

keep the sub-features as features to be enabled